### PR TITLE
Elastic: add a configuration for monitoring.ui.ccs.enabled.

### PR DESCRIPTION
**Pull request by micedre**
When false, allow to enable stack monitoring in kibana.

https://www.elastic.co/guide/en/kibana/current/monitoring-settings-kb.html

{CCDB3FF2-D703-44BD-B03B-76311E78C999}

⚠️ Pas vraiment testé

### CODE REVIEW

**Comment by ihiverlet** [comment]
Hello, 
I'm having a hard time trying to get this "cluster overview". Can you be more precise on what's expected please ? Especially where should I see this "cluster overview" ?

**Comment by Gaspi** [comment]
**TL;DR**: The PR is ok to me and only needs a [`Chart.yaml`](https://github.com/micedre/helm-charts-databases/blob/main/charts/elastic/Chart.yaml#L31) version bump to be merged. Thx!

My understanding is that it is rather a matter of **disabling** the cross-cluster monitoring which is enabled by default (`monitoring.ui.ccs.enabled: true`) and prevents the dashboard in @micedre 's snapshot from properly showing up (cf: https://github.com/elastic/kibana/issues/125756).

With `monitoring.ui.ccs.enabled: false` added to Kibana's `extraConfiguration`, the dashboard can indeed be found in Management > Stack Monitoring.

Cross-cluster monitoring is probably a bit too hard to setup so that we could have left a default `false` in `values.yaml` without bothering describing the field in the `values.schema.json`.

I see that `xpack.monitoring.collection.enabled: true` also needs to be set on the elasticsearch side (`elasticsearch.extraConfig`) but kibana already takes care of that the first time we connect to the dashboard so we probably don't need to bother.
